tests/main/views/test_services.py

```diff
@@ -218,11 +218,12 @@ def test_enabled_service_has_unavailable_banner(self):
         assert unavailable_banner.exists
         assert unavailable_banner.heading_text() == '{} stopped offering this service on {}'.format(
             self.service['services']['supplierName'],
-            'Tuesday 5 January 2016.',
+            'Thursday 6 January 2000.',
         )
         assert unavailable_banner.body_text() == 'Any existing contracts for this service are still valid.'
 
     def test_disabled_service_has_unavailable_banner(self):
+        self.service.frameworkName
```

Comment:
I think this might have been c&p-ed from somewhere?

Reply:
🤦  good spot! Have removed in https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/1050/commits/c52a9fce1342b30a24f7c5d5c96b596ce82110ea